enhancements/update/phased-rollouts.md

```diff
@@ -0,0 +1,242 @@
+---
+title: phased-update-rollouts
+authors:
+  - "@wking"
+reviewers:
+  - "@LalatenduMohanty"
+  - "@sdodson"
+  - "@vrutkovs"
+approvers:
+  - TBD
+creation-date: 2020-07-20
+last-updated: 2020-07-20
+status: implementable
+---
+
+# Phased Update Rollouts
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement request proposes phased update rollouts, allowing release administrators to declare a time window over which update recommendations are phased into per-cluster requets for stable channel [Cincinnati graphs][cincinnati].
+
+## Motivation
+
+We [currently][channel-semantics] has `fast-4.y` and `stable-4.y` channels for each 4.y series.
+That allows users who want to hear about releases and related update edges immediately after we declare them supported to do so via `fast-4.y`, while users who want to wait and see if we still recommend the edge after its seen more volume can do so via `stable-4.y`, as discussed [here][fast-vs-stable].
+However, clusters default to `stable-4.y` and users need to explicitly configure `fast-4.y` to populate that higher-risk cluster pool.
+This leaves the additional safety of `stable-4.y` ambiguous; when a cluster sees an available update recommended in `stable-4.y`, it's not clear now much traffic it has received.
+Conservative users are left to wait an arbitrary amount of additional time and guess about the number of other clusters which may be feeling out the new edge, or rely entirely on their own local testing.
+
+By smearing the update recommendation out over a time window, we address several issues:
+
+1. Reduced likelihood of a thundering herd of updating clusters swamping canonical image registries.
+    We currently lack built-in support for [automatic updates][automatic-updates], but some users have built analogous functionality as a higher-level driver for ClusterVersion.
+    Phased rollouts make it less likely to have many clusters simultaneously initiate a given update by removing the synchronized update recommendation.
```

Comment:
In long term we will see many clusters simultaneously update even with phased rollout(when our customer base grows). However the difference is , with phased rollout majority of the overall cluster fleet can not update at the same time.

Reply:
I'm arguing that this is part of spreading the load on canonical image registries here, not trying to argue that there will be no partial overlaps between multiple clusters.  But is there particular wording you think I should add or adjust to make that more clear?

Reply:
"Phased rollouts reduces the number of clusters able to simultaneously initiate a given update by removing the synchronized update recommendation."